resources/views/community/components/user/profile/last-seen-in.blade.php

```diff
@@ -0,0 +1,64 @@
+@props([
+    'userMassData' => [],
+])
+
+<?php
+use App\Platform\Models\PlayerSession;
+use App\Platform\Models\Game;
+use Illuminate\Support\Carbon;
+
+$mostRecentSession = PlayerSession::where('user_id', $userMassData['ID'])
+    ->with('game')
+    ->orderBy('created_at', 'desc')
+    ->first();
+
+// If there's no session for some reason, try to fall back to the user record.
+$sessionGame = $mostRecentSession?->game;
+if (!$mostRecentSession) {
+    $sessionGame = Game::with('system')->find($userMassData['LastGame']['ID']);
+}
+
+$mostRecentRichPresenceMessage = (
+    $mostRecentSession?->rich_presence
+    ?? $userMassData['RichPresenceMsg']
+    ?? null
+);
+
+$parsedDate = Carbon::parse($mostRecentSession->rich_presence_updated_at);
```

Comment:
this needs a `?->` too. Try visiting an account that hasn't been used in a while - like Brian.

Reply:
Thanks, addressed in latest.